cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -104,6 +106,7 @@ def run_cmd(self, cmd):
             return
         output = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
         click.echo(output.decode('utf-8'))
+        return output
```

Comment:
There the function's result is used?
I don't see any application.

Reply:
Yeah, it was used in the previous commit. Now it's not.